llama_cloud_services/extract/extract.py

```diff
@@ -44,6 +51,17 @@
 )
 
 
+def _is_retryable_error(exception: BaseException) -> bool:
+    """Check if an exception is retryable."""
+    if isinstance(exception, ApiError):
+        return exception.status_code in (502, 503, 504, 425, 408)
```

Comment:
429?

Reply:
Just to keep it safe and avoid additional requests if the jobs service endpoint is getting hammered already.